src/global-firestore-data/courses.ts

```diff
@@ -0,0 +1,22 @@
+import { doc, getDoc } from 'firebase/firestore';
+import { coursesCollection, availableRostersForCoursesCollection } from '../firebase-config';
+
+export const getCourse = async (roster: string, subject: string, number: string) => {
```

Comment:
Can we add a doc comment for this and annotate its return type?

Reply:
Yerp